### PR TITLE
OSDOCS-3679: Adding steps to change the mode to automatic to actually…

### PR DESCRIPTION
**Pull request by bergerhoffer**
… evict pods

Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3679

Link to docs preview:
http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3679/nodes/scheduling/nodes-descheduler.html#nodes-descheduler-installing_nodes-descheduler

Additional information:

### CODE REVIEW

**Comment by ingvagabund** [comment]
LGTM

File: modules/nodes-descheduler-installing.adoc [hunk 1]
```diff
@@ -43,6 +45,8 @@ endif::[]
 .. From the *Operators* -> *Installed Operators* page, click the *Kube Descheduler Operator*.
 .. Select the *Kube Descheduler* tab and click *Create KubeDescheduler*.
 .. Edit the settings as necessary.
+... To evict pods instead of simulating the evictions, change the *Mode* field from *Predictive* to *Automatic*.
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/nodes-descheduler-installing.adoc:48]: good suggestion, thanks!

**Pull request merged**